src/nomad_analysis/__init__.py

```diff
@@ -1,20 +1,4 @@
-#
```

Comment:
do we need this init file in the root? I generally do not use it

Reply:
No, this is only needed to have the version of the package also loaded when the package is imported. 
That is, to make this work 
```py
import nomad_analysis
print(nomad_analysis.__version__)
```

Reply:
my bad, it should be there